dowhy/causal_estimator.py

```diff
@@ -126,6 +182,48 @@ def do(self, x):
     def construct_symbolic_estimator(self, estimand):
         raise NotImplementedError
 
+    def get_confidence_intervals(self, estimate):
+        '''
+            Find the confidence intervals corresponding to any estimator
+            This is done with the help of bootstrapped confidence intervals
+
+            For more details, refer to the following link:
+            https://ocw.mit.edu/courses/mathematics/18-05-introduction-to-probability-and-statistics-spring-2014/readings/MIT18_05S14_Reading24.pdf
+        '''
+
+        # The array that stores the results of all estimations
+        simulation_results = np.zeros(self.num_ci_simulations)
+
+        # Find the sample size the praportion with the population size
+        sample_size = int( self.sample_size * len(self._data) )
+
+        if sample_size > len(self._data):
+            self.logger.warning("WARN: The sample size is greater than the population being sampled")
```

Comment:
wording: it's not greater than the "population", but greater than the dataset provided. Typically, the population contains all possible infinite data points while a dataset is a sample from it.